tests/services/test_criteria_matcher.py

```
@@ -0,0 +1,326 @@
+
+    def _mock_file(filename="test.txt", **stat_kwargs):
+        file_path = tmp_path / filename
+        file_path.touch()
+        stat_info = mock_stat(**stat_kwargs)
+        
+        # Patch Path.stat to return our mock stat_info
+        path_stat_patcher = patch.object(Path, "stat", return_value=stat_info)
+        # Also patch os.stat for consistency if called directly
+        os_stat_patcher = patch("os.stat", return_value=stat_info)
+        
+        path_stat_mock = path_stat_patcher.start()
```

Ruff (0.14.14)

[error] Local variable `path_stat_mock` is assigned to but never used

Remove assignment to unused variable `path_stat_mock`

(F841)